src/main/java/com/example/springboot/controller/ReaderController.java

```diff
@@ -0,0 +1,90 @@
+package com.example.springboot.controller;
+
+import java.io.IOException;
+import java.util.Collection;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.springboot.base.Reader;
+import com.example.springboot.dao.DAOInterface;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * This is class that control work of application, process requests where URL start with '/reader'
+ * @author Dmytro Kohut
+ * @version 1.1
+ */
+@RestController
+@RequestMapping("/reader")
+public class ReaderController {
+	
+	@Autowired
+	private DAOInterface<Reader> readerDAO;
+	
+	static final private ObjectMapper objectMapper = new ObjectMapper();
+	
+	/**
+	 * Get information about all readers
+	 * @return
+	 */
+	@RequestMapping(value="/all", method=RequestMethod.GET)
+	public Collection<Reader> findAllReaders(){
+		return readerDAO.findAll();
```

Comment:
WTF? You have started using Spring REST, so please use Spring response classes for it